front/app/api/event_images/__mocks__/useEventImage.ts

```diff
@@ -1,67 +1,20 @@
-export const eventImageData = [
-  {
-    id: '30c1b604-71fd-4ac4-9cd7-3e5601d9cb0f',
-    type: 'image',
-    attributes: {
-      ordering: null,
-      created_at: '2023-02-28T05:56:37.762Z',
-      updated_at: '2023-02-28T05:56:37.762Z',
-      versions: {
-        small: 'http://localhost:4000/uploads/small.png',
-        medium: 'http://localhost:4000/uploads/medium.png',
-        large: 'http://localhost:4000/uploads/large.png',
-        fb: 'http://localhost:4000/uploads/fb.png',
-      },
-    },
-  },
-];
+import { IEventImageData } from '../types';
 
-export const eventData = {
```

Comment:
We had eventData in this file because we need event_images to be defined in the event's relationships. I have now created that relationship in the useEventImage.test.ts file instead, which is clearer. We also shouldn't create mocks for other endpoints in this mock file.